test/utils/timestamp.test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+
+const {format} = require('../../lib/utils/timestamp');
+const {config, override} = require('../../config');
+const timestampOff = require('../../lib/constants/timestamp').none;
+
+const cachedConfig = {...config};
+
+describe('timestamp util', () => {
+	after(() => {
+		override(cachedConfig);
+	});
+
+	it('should format date correctly', async() => {
+		override({timestamp: 'MMMM DD, YYYY'});
```

Comment:
This sort of things make our unit tests unstable. We should rewrite API in a way that it's testable and there is no need to override global configs for this.
One option would be to rewrite "format" function to accept date format as parameter.
Other option - curry it and save in some variable.
e.g. `const format = format => date => {...}`
and then somewhere do `const formatDate = format(config.format)`. You get the idea ;)